test/cli/file-table-json/test.sh

```diff
@@ -7,6 +7,6 @@ main/sorbet --censor-for-snapshot-tests --silence-dev-message -p file-table-json
 echo "----- --no-stdlib -p file-table-json -----------------------------------"
 main/sorbet --censor-for-snapshot-tests --silence-dev-message --no-stdlib -p file-table-json test/cli/file-table-json/file-table-json.rb
 echo "----- -p file-table-full-json ------------------------------------------"
-main/sorbet --censor-for-snapshot-tests --silence-dev-message -p file-table-full-json test/cli/file-table-json/file-table-json.rb
+main/sorbet --censor-for-snapshot-tests --silence-dev-message -p file-table-full-json test/cli/file-table-json/file-table-json.rb | sed 's/https:\/\/.*procs.rbi/procs.rbi/g'
```

Comment:
Output directory changed from `bazel-out/host/` to `bazel-out/k8-opt-exec-2B5CBBC6` and is platform dependent.

I'm sure there is a smarter way to do this?

Reply:
What do you think about the approach I had taken before?

https://github.com/sorbet/sorbet/pull/7509/commits/fb5f06c764014e5571bb71c78baafdc68a136ff6

Reply:
Since this file ~never changes, it will have the effect of making it easier for people to grep and find this RBI file, since it will be checked into the repo. That commit also adds a test to make sure that the version checked into the repo matches the one produced by the bazel job.

Reply:
I pulled your approach in https://github.com/sorbet/sorbet/pull/7827/commits/d1d2fccc5b3c28e198a8ea4e317ea23c6e713d87 👍